src/app/campaign.service.ts

```diff
@@ -190,6 +191,11 @@ export class CampaignService {
     return of([
     ]);
   }
+
+  getCampaignImpactStats(): Observable<CampaignStats>{
+    return of(new CampaignStats(1000, 5));
```

Comment:
Doesn't this need to be as below? These are the numbers I see now at https://donate.biggive.org/
```suggestion
    return of(new CampaignStats(242_309_804, 15_000));
```